guides/source/active_job_basics.md

```diff
@@ -406,6 +466,30 @@ end
 [`around_perform`]: https://api.rubyonrails.org/classes/ActiveJob/Callbacks/ClassMethods.html#method-i-around_perform
 [`after_perform`]: https://api.rubyonrails.org/classes/ActiveJob/Callbacks/ClassMethods.html#method-i-after_perform
 
+### Callbacks with Bulk Enqueuing
+
+When enqueueing jobs in bulk using `perform_all_later`, callbacks such as `around_enqueue`, will not be triggered on the individual jobs.
+
+ However, the `perform_all_later` method does fire an [`enqueue_all.active_job`][] event which you can subscribe to using `ActiveSupport::Notifications`. 
+
+The event payload contains the adapter name, enqueued jobs count, the list of job instances, etc. `successfully_enqueued?` can be used to find out if a given job was successfully enqueued. 
```

Comment:
(non-blocking, suggestion): I didn't realize that the sentence ended with etc. and sho I tried to read both these sentences as one. It felt like it impacted the readability because the new sentence did not start with a capital letter. 

```suggestion
The event payload contains the adapter name, enqueued jobs count, the list of job instances, etc. A method `successfully_enqueued?` can be used to find out if a given job was successfully enqueued. 
```

Reply:
Maybe we can also provide a link to the `successfully_enqueued?` method?

Reply:
Sure. I've linked it though that api is pretty bare https://api.rubyonrails.org/classes/ActiveJob/Core.html#method-i-successfully_enqueued-3F